tests/connector-service-tests/test/testkit/connectorclient.go

```diff
@@ -126,8 +146,13 @@ func getRequestWithHeaders(t *testing.T, url string, headers map[string]string)
 func parseErrorResponse(t *testing.T, response *http.Response) *Error {
 	logResponse(t, response)
 	errorResponse := ErrorResponse{}
-	err := json.NewDecoder(response.Body).Decode(&errorResponse)
-	require.NoError(t, err)
+	decoder := json.NewDecoder(response.Body)
+
+	if decoder.More() {
+		err := decoder.Decode(&errorResponse)
```

Comment:
I do not think we need to check for more, as we do not expect multiple error responses.